cdp_scrapers/tests/scraper_tests.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from cdp_scrapers.instances.seattle import SeattleScraper
+from cdp_scrapers.instances.kingcounty import KingCountyScraper
+from datetime import datetime
+
+
+@pytest.mark.parametrize(
+    "start_date_time, end_date_time, number_of_events,"
+    "number_of_sessions_in_first_event, number_of_event_minute_items,"
+    "event_with_votes, event_minute_item_with_votes, number_of_votes,"
+    "expected_caption_uri_in_first_session, expected_video_uri_in_first_session",
+    [
+        # Check for 1 event with 1 session and 10 event minute items.
```

Comment:
To me, these first 2 tests don't gain us anything that the latter 2 do not.
This first one tests for multiple event minute items. Well, the last one tests for that as well, along with votes.
The second test checks getting multiple events from a period of time, which again the last test does.
So I feel like the first 2 tests are kind of meaningless. Am I missing something Shak? I feel like tests 3 and 4 test for everything the first 2 do, and then more, like number of sessions and votes.

Reply:
I removed the first 2 Seattle tests now.